src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/MIReplicaReportingServiceTest.java

```diff
@@ -1393,4 +1491,27 @@ private List<Timestamp> callGetReportRefreshRequestTimes(List<String> taskIdList
             return Collections.emptyList();
         }
     }
+
+    void callRefreshReportTasks(int refreshRecordsCount) {
+        log.info(String.format("callRefreshReportTasks with maxRecordsCount : %s", refreshRecordsCount));
+
+        String runFunction = " call cft_task_db.refresh_report_tasks( ? ) ";
+
+        try (Connection conn = DriverManager.getConnection(
+            containerReplica.getJdbcUrl(), containerReplica.getUsername(), containerReplica.getPassword());
+             PreparedStatement preparedStatement = conn.prepareStatement(runFunction)) {
+
+            preparedStatement.setInt(1, refreshRecordsCount);
+            preparedStatement.execute();
+
+        } catch (SQLException e) {
+            log.error("Procedure call refresh_report_tasks failed with SQL State : {}, {} ",
+                      e.getSQLState(), e.getMessage());
+        } catch (Exception e) {
+            log.error("Procedure call refresh_report_tasks failed with SQL State : {}, {} ",
+                      e.getCause(), e.getMessage());
+        }
```

Comment:
These exceptions should make the test fail rather than just logging it out.  Should rethrow the exception after logging.

Reply:
Addressed it.